pkgs/applications/misc/pipr/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, bubblewrap
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "pipr";
+  version = "v0.0.12";
+
+  src = fetchFromGitHub {
+    owner = "ElKowar";
+    repo = pname;
+    rev = "v0.0.12";
+    sha256 = "0l7yvpc59mbzh87lngj6pj8w586fwa07829l5x9mmxnkf6srapmc";
+  };
+
+  cargoSha256 = "1gbzi4bx6chf8hpxbf99bnncdgi2xqj41fh9jldwz53rnfvwvayn";
+
+  buildInputs = [ bubblewrap ];
+
+  meta = with stdenv.lib; {
+    description = "A commandline-tool to interactively write shell pipelines";
+    homepage = "https://github.com/ElKowar/pipr";
+    license = licenses.mit;
+    maintainers = maintainers.elkowar;
```

```suggestion
    maintainers = with maintainers; [ elkowar ];
```

This should be a list.